### PR TITLE
Add tts document&fix bug

### PR DESCRIPTION
**Pull request by charli117**

### CODE REVIEW

**Review by crazywoola — DISMISSED** [review]

File: api/core/model_runtime/docs/zh_Hans/schema.md [hunk 1]
```diff
@@ -59,6 +63,7 @@
 - `text-embedding` 文本 Embedding 模型
 - `rerank` Rerank 模型
 - `speech2text` 语音转文字
+- `text2speech` 文字转语音
```
Comment (takatost) [inline api/core/model_runtime/docs/zh_Hans/schema.md]: Looks like this needs to be converted to `tts` here as well

File: api/core/model_runtime/docs/en_US/schema.md [hunk 2]
```diff
@@ -58,6 +62,7 @@
 - `text-embedding` Text Embedding model
 - `rerank` Rerank model
 - `speech2text` Speech to text
+- `text2speech` Text to speech
```
Comment (takatost) [inline api/core/model_runtime/docs/en_US/schema.md]: Looks like this needs to be converted to `tts` here as well

File: api/core/model_runtime/docs/zh_Hans/images/index/image-20231210143654461.png
Comment (takatost) [inline api/core/model_runtime/docs/zh_Hans/images/index/image-20231210143654461.png:1]: You can change the image back. I'll retake a new one later.

File: api/core/model_runtime/docs/en_US/images/index/image-20231210143654461.png
Comment (takatost) [inline api/core/model_runtime/docs/en_US/images/index/image-20231210143654461.png:1]: You can change the image back. I'll retake a new one later.

**Review by takatost — CHANGES_REQUESTED** [review]

File: api/core/model_runtime/docs/en_US/images/index/image-20231210143654461.png
Comment (charli117) [reply api/core/model_runtime/docs/en_US/images/index/image-20231210143654461.png:1]: > You can change the image back. I'll retake a new one later.

@takatost model name have been modified, image also restored

**Review by takatost — APPROVED** [review]
lgtm

**Pull request merged**